Rubberduck.Parsing/ComReflection/ComParameter.cs

```diff
@@ -87,48 +93,60 @@ private void GetParameterType(TYPEDESC desc, ITypeInfo info)
             var vt = (VarEnum)desc.vt;
             TYPEDESC tdesc;
 
-            switch (vt)
+            if (vt == VarEnum.VT_PTR)
             {
-                case VarEnum.VT_PTR:
-                    tdesc = (TYPEDESC)Marshal.PtrToStructure(desc.lpValue, typeof(TYPEDESC));
-                    GetParameterType(tdesc, info);
-                    IsByRef = true;
-                    break;
-                case VarEnum.VT_USERDEFINED:
-                    int href;
-                    unchecked
-                    {
-                        href = (int)(desc.lpValue.ToInt64() & 0xFFFFFFFF);
-                    }
-                    try
+                tdesc = Marshal.PtrToStructure<TYPEDESC>(desc.lpValue);
+                GetParameterType(tdesc, info);
+                IsByRef = true;
+            }
+            else if (vt == VarEnum.VT_USERDEFINED)
+            {
+                int href;
+                unchecked
+                {
+                    href = (int)(desc.lpValue.ToInt64() & 0xFFFFFFFF);
+                }
+
+                try
+                {
+                    info.GetRefTypeInfo(href, out ITypeInfo refTypeInfo);
+                    refTypeInfo.GetTypeAttr(out IntPtr attribPtr);
+                    using (DisposalActionContainer.Create(attribPtr, refTypeInfo.ReleaseTypeAttr))
                     {
-                        info.GetRefTypeInfo(href, out ITypeInfo refTypeInfo);
-                        refTypeInfo.GetTypeAttr(out IntPtr attribPtr);
-                        using (DisposalActionContainer.Create(attribPtr, refTypeInfo.ReleaseTypeAttr))
+                        var attribs = Marshal.PtrToStructure<TYPEATTR>(attribPtr);
+                        var type = new ComDocumentation(refTypeInfo, -1).Name;
```

Comment:
Thinking we should avoid magical number and use a constant `MEMBER_NIL` (or whatever is it that `-1` should be).

Reply:
Good idea. It doesn't really have a name that I could find in the documentation, so I just called it `ComDocumentation.LibraryIndex`. Save caveat about the sync of unrelated crap.